package.json

```diff
@@ -13,6 +13,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-icons": "^4.3.1",
+    "react-medium-image-zoom": "^4.3.5",
```

Comment:
What are we using this for?

Reply:
When you click on an image it will zoom the image. I made it with this package.